pkg/sdkserver/sdkserver.go

```diff
@@ -69,6 +71,23 @@ var (
 	_ beta.SDKServer  = &SDKServer{}
 )
 
+type counterUpdateRequest struct {
+	// Tracks CountIncrement and / or CountDecrement Requests
+	diffList []int64
+	// Counter (as retreived during first request)
+	counter agonesv1.CounterStatus
+	// Tracks if there have been any CountSet requests
+	countSet bool
+	// Tracks if there have been any CapacitySet requests
+	capacitySet bool
```

Comment:
Ha no, the gRPC standard when wildly out the door with this change. (It would work if we were doing webhook validation :)) This was mainly from a comment months ago on not doing nilable pointers, but having nilable pointers does make sense in the context.